src/cascadia/TerminalControl/TermControlAutomationPeer.cpp

```diff
@@ -28,6 +28,32 @@ namespace XamlAutomation
     using winrt::Windows::UI::Xaml::Automation::Provider::ITextRangeProvider;
 }
 
+static constexpr wchar_t UNICODE_NEWLINE{ L'\n' };
+
+static inline std::wstring Sanitize(std::wstring_view text)
+{
+    std::wstring sanitized;
+    std::for_each(text.begin(), text.end(), [&sanitized](auto&& c) {
+        if (c >= UNICODE_SPACE || c == UNICODE_NEWLINE)
+        {
+            sanitized += c;
+        }
+    });
+    return sanitized;
+}
+
+static constexpr bool IsReadable(std::wstring_view text)
+{
+    for (const auto c : text)
+    {
+        if (c > UNICODE_SPACE)
+        {
+            return true;
+        }
+    }
+    return false;

```

Comment:
```cpp
return text.find(UNICODE_SPACE) != text.end();
```

Reply:
> `return text.find(UNICODE_SPACE) != std::wstring_view::npos;`

wait, isn't that different though? I'm trying to find a character `c > UNICODE_SPACE`. So would this work:
```
    return std::find_if(text.begin(), text.end(), [](wchar_t c) {
               return c > UNICODE_SPACE;
           }) != std::wstring_view::npos;
```

Reply:
Oh woops. In that case the loop is probably easier to understand than the complex `std::find_if`.